server/models/TrackReceipt.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+
+const trackReceiptSchema = new mongoose.Schema({
```

Comment:
trackReceipt is not the right name for this object. This is more of an 'addedSong.'